docs/en/engines/table-engines/mergetree-family/replication.md

```diff
@@ -304,6 +304,12 @@ We use the term `MergeTree` to refer to all table engines in the `MergeTree fami
 
 If you had a `MergeTree` table that was manually replicated, you can convert it to a replicated table. You might need to do this if you have already collected a large amount of data in a `MergeTree` table and now you want to enable replication.
 
+`MergeTree` table can be automatically converted on server restart if `convert_to_replicated` flag is set at the table's flags directory (`/var/lib/clickhouse/data/db_name/table_name/flags/`). 
```

Comment:
I like this approach better, but the disadvantage is that user can't remove the flag without a working server. 
For example, if you set `convert_to_replicated` setting for a table with an `Ordinary` database, the `NOT_IMPLEMENTED` exception is raised when the server is loaded and it becomes impossible to start the server. 
In this case, you can either prohibit setting this setting in the `ALTER` query interpreter (but it helps only in this case), or remove the setting when an exception occurs and load without conversion.

Reply:
Hm, yep, it's a good point. But I don't like `table_name/flags/` dir either because we have some code that doesn't expect to find extra subdirs in the table dir, for example https://github.com/ClickHouse/ClickHouse/blob/852f397a97a8af0aecf8408cde065eb0dd99a52d/src/Storages/MergeTree/MergeTreeData.cpp#L2779-L2800 

But maybe it's okay if we ensure that it's removed on server start after the coversion. Also, I would prefer a single flag file, without the subdir